contracts/mocks/MemberRoles/MRMockCover.sol

```diff
@@ -32,5 +35,7 @@ contract MRMockCover {
   function createMockCover(address to, uint tokenId) public {
     coverNFT.mint(to, tokenId);
   }
-
+  function switchMembershipOf(address member, address newAddress) external {
+    memberRoles.switchMembershipOf(member, newAddress);
+  }
```

Comment:
Cover contract doesn't have this function. Why's this here?

Reply:
`switchMembershipOf` is onlyInternal function in the MemberRoles contract this is being used for testing of that function